docs/workflow/building/libraries/README.md

```diff
@@ -238,11 +238,11 @@ To build a library's package, simply invoke `dotnet pack` on the src project aft
 
 ```cmd
 build libs
-dotnet pack src\libraries\System.Text.Json\src\
+.dotnet/dotnet pack src\libraries\System.Text.Json\src\
```

Comment:
There's a top-level dotnet.sh/cmd script. The `.dotnet` directory is implementation detail.

Reply:
Aah. I was working in PowerShell and that did not work for me. Maybe would be good to mention that and write like that `dotnet.cmd/.sh pack src\libraries\System.Text.Json\src\` ?

Reply:
It's automatically picked up in cmd but not in powershell. The above command works in most cases but not all. Feel free to change it to dotnet.cmd/sh